plugin/highlight/plugin.js

```diff
@@ -71,15 +72,15 @@ const Plugin = {
 
 		// Run initial highlighting for all code
 		if( config.highlightOnLoad ) {
-			Array.from( reveal.getRevealElement().querySelectorAll( 'pre code' ) ).forEach( block => {
+			Array.from( reveal.getRevealElement().querySelectorAll( config.cssSelector ) ).forEach( block => {
 				Plugin.highlightBlock( block );
 			} );
 		}
 
 		// If we're printing to PDF, scroll the code highlights of
 		// all blocks in the deck into view at once
 		reveal.on( 'pdf-ready', function() {
-			[].slice.call( reveal.getRevealElement().querySelectorAll( 'pre code[data-line-numbers].current-fragment' ) ).forEach( function( block ) {
+			[].slice.call( reveal.getRevealElement().querySelectorAll( config.cssSelector + '[data-line-numbers].current-fragment' ) ).forEach( function( block ) {
```

Comment:
This line makes sense only for compatible modifications to the selector.
If you wanted to give complete control to the user, a new configuration would need to be added. Something like
```js
config.cssSelectorLineNumbers = typeof config.cssSelectorLineNumbers === 'string' ? config.cssSelectorLineNumbers : 'pre code[data-line-numbers].current-fragment';
```